packages/xchain-thorchain/__e2e__/thorchain-client.integration.ts

```diff
@@ -0,0 +1,71 @@
+import { Client as BnbClient } from '@xchainjs/xchain-binance'
```

Comment:
[nitpick] Rename `thorchain-client.integration.ts` to `thorchain-client.e2e.ts' or `thorchain-client.test.ts`

Reply:
done